app/terminal/terminal.tsx

```diff
@@ -296,25 +296,8 @@ export default class TerminalComponent extends React.Component<TerminalProps, St
     this.updateLineLengthLimit();
   }
   private onDownloadClick() {
-    const serveLog = (log: string) => {
-      const element = document.createElement("a");
-      const plaintext = toPlainText(log);
-      element.setAttribute("href", "data:text/plain;charset=utf-8," + encodeURIComponent(plaintext));
-      element.setAttribute("download", "build_logs.txt");
-      element.click();
-    };
-    if (this.props.fullLogsFetcher) {
-      this.setState({ isLoadingFullLog: true });
-      this.props
-        .fullLogsFetcher()
-        .then(serveLog)
-        .catch((e) => errorService.handleError(e))
-        .finally(() => {
-          this.setState({ isLoadingFullLog: false });
-        });
-      return;
-    }
-    serveLog(this.props.value);
+    event.preventDefault();
```

Comment:
Regular `<button>` elements don't have a default action when clicked, so the `event.preventDefault()` here is essentially a nop.

(An example where `preventDefault()` would be needed is if you had an `<a href="...">` and you wanted to prevent the browser's default behavior of navigating to the URL, and instead handle the link click with custom application logic.)

Reply:
Ooooooh, thanks for explaining.